sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -373,17 +374,18 @@ def stage_job_resources(
 
   if hasattr(setup_options, 'sdk_location'):
     if setup_options.sdk_location == 'default':
-      stage_tarball_from_remote_location = True
+      stage_sdk_from_remote_location = True
     elif (setup_options.sdk_location.startswith('gs://') or
           setup_options.sdk_location.startswith('http://') or
           setup_options.sdk_location.startswith('https://')):
-      stage_tarball_from_remote_location = True
+      stage_sdk_from_remote_location = True
     else:
-      stage_tarball_from_remote_location = False
+      stage_sdk_from_remote_location = False
 
-    staged_path = FileSystems.join(google_cloud_options.staging_location,
-                                   names.DATAFLOW_SDK_TARBALL_FILE)
-    if stage_tarball_from_remote_location:
+    staged_path = FileSystems.join(
```

Comment:
Should `staged_path` be moved inside the `else`? I do not see it being used outside that branch.

Reply:
Thanks, I like that.